styles.css

```diff
@@ -206,6 +207,7 @@ nav {
 
 .about .about_details {
   color: #344563;
+  font-size: 18px;
```

Comment:
Increased font size in the about section for better readability in the mobile version.